cfgov/v1/tests/handlers/blocks/test_feedback.py

```diff
@@ -25,6 +28,9 @@ def setUp(self):
                                         'radio_intro': None}
         )
 
+    # I'm not even sure how this is going to work with Python3's
+    # unicode-by-default approach. Punting.
+    @skipIf(six.PY3, 'we need to rework the encoding handling for Python 3')
```

Comment:
This is me completely punting — I have no idea how the attempts to handle non-ascii referrer URLs are going to work when we're actually running in Python 3. This will need work once we have the site actually running.